lessons/lesson_6/Lesson_6_Home_Work_conditions.py

```diff
@@ -5,10 +5,35 @@
 Steps:
 
 Open the web application in a browser.
-Log into the application with provided credentials.
+Log into the application with provided credentials.1
 From the side menu, navigate to the "HR Administration" section.
 Click on the "Add User" button.
 Locate the user status option on the "Add User" form.
 Check the current status of the user. If it is not "Disabled," you need to select the "Disabled" option.
 Ensure your script can handle both conditions: if the status is already "Disabled," it remains unchanged; otherwise, change it to "Disabled."
-'''
\ No newline at end of file
+'''
+
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+import time
+
+driver = webdriver.Chrome()
+driver.get("https://portnov_admin-trials711.orangehrmlive.com/client/#/dashboard")
+
+driver.find_element(By.CSS_SELECTOR, "input[id='txtUsername']").send_keys("Admin")
+driver.find_element(By.CSS_SELECTOR, "input[id='txtPassword']").send_keys("qTJn5@5APu")
+driver.find_element(By.CSS_SELECTOR, "button[type='submit']").click()
+
+time.sleep(5)
+driver.find_element(By.CSS_SELECTOR, 'div[class="dashboard-widget-config-button"]').click()
+time.sleep(5)
+buttons = driver.find_elements(By.CSS_SELECTOR, 'div[class="configuration-tabs"] input[id*="adminWidgetSwitch"]')
+
+for item in buttons:
```

Comment:
It looks like you just copied the code from Lesson_6_conditions_demonstration.py I can't see actions from the assignment.